src/utils/command-helpers.mjs

```diff
@@ -227,12 +227,17 @@ const DEBOUNCE_WAIT = 100
  * @param {string | string[]} target
  * @param {Object} opts
  * @param {number} [opts.depth]
+ * @param {Array<string|RegExp>} [opts.ignored]
  * @param {() => any} [opts.onAdd]
  * @param {() => any} [opts.onChange]
  * @param {() => any} [opts.onUnlink]
  */
-export const watchDebounced = async (target, { depth, onAdd = () => {}, onChange = () => {}, onUnlink = () => {} }) => {
-  const watcher = chokidar.watch(target, { depth, ignored: /node_modules/, ignoreInitial: true })
+export const watchDebounced = async (
+  target,
+  { depth, ignored = [], onAdd = () => {}, onChange = () => {}, onUnlink = () => {} },
+) => {
+  const baseIgnores = [/\/(node_modules|.git)\//]
```

Comment:
We were previously ignoring `node_modules` only. I've added `.git` because setting up watchers there is also wasteful.